lms/djangoapps/certificates/models.py

```diff
@@ -304,20 +304,17 @@ def certificate_info_for_user(user, course_id, grade, user_is_whitelisted=None):
             user=user, course_id=course_id, whitelist=True
         ).exists()
 
+    user_is_eligible, certificate_is_delivered, certificate_type = 'N', 'N', 'N/A'
```

Comment:
I know this is perfectly valid code, but I'd rather see each variable declared on its own line